ENG-2465 - Add New Login API responses and fields (#3603)
FusionAuth/fusionauth-site: astro/src/content/docs/apis/_login-response-codes.mdx

| 203  | The user was authenticated successfully. The user is required to change their password, the response will contain the `changePasswordId` to be used on the [Change Password](/docs/apis/users#change-a-users-password) API.  Since version `1.15.0`, the response will also contain a `changePasswordReason` field which can have one of the following values: <ul><li>`Administrative` - An administrator has required this user to change their password.</li><li>`Breached` - The password has been found to have belonged to a breached dataset and must be changed, per the Reactor configuration.</li><li>`Expired` - The password has expired, per the expiration setting in the tenant configuration.</li><li>`Validation` - The password fails the validation rules configured in the tenant password settings.</li></ul><JSON title="Example Response JSON" src="login/login-change-password-response.json" /> |

> | 212  | The user's email address (if an email address or username was used to authenticate) or phone number has not yet been verified. The response will contain the User object that was authenticated. If the tenant verification strategy has been set to `FormField`, the response will contain the verification Id that was generated for the user.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |

Reviewer: If you like this, apply it to _login-response-codes-authenticated.mdx, too.

```suggestion
| 212  | Returned either when the user's email address or phone number has not yet been verified, or when the user authenticated with a username, which is not verifiable. The response will contain the User object that was authenticated. If the tenant verification strategy has been set to `FormField`, the response will contain the verification Id that was generated for the user.  
|
```

Author: Not sure. The revision almost makes it sounds like all usernames will result in a 212, which is not the case. It would only happen when a username is used AND the email is unverified.

Reviewer: Got it, makes sense when you say it that way. I think it's just hard to put ((email_auth OR username_auth) AND email_unverified) OR (phone_auth AND phone_unverified) into English.
